### PR TITLE
Fix for including a screenshot bug on Windows (#124)

### PR DESCRIPTION
**Pull request by pinkie1378**

### CODE REVIEW

**Comment by nicoddemus** [comment]
LGTM, the failure seems unrelated though.

**Comment by pinkie1378** [comment]
The Node.js tests didn't run because of a [problem during PhantomJS installation](https://travis-ci.org/pytest-dev/pytest-html/jobs/299359717). This is unrelated to the proposed change. Please advise.

**Comment by nicoddemus** [comment]
@pinkie1378 I don't there's anything to be done in this PR, let's wait for @davehunt's input (he will be back next week it seems).

File: pytest_html/plugin.py [hunk 1]
```diff
@@ -159,8 +160,14 @@ def append_extra_html(self, extra, extra_index, test_index):
...
-                if content.startswith(('file', 'http')) or \
-                        os.path.isfile(content):
+                try:
+                    is_file = (content.startswith(('file', 'http')) or
+                               isfile(content))
+                except ValueError:
+                    # On Windows, os.path.isfile throws this exception when
+                    # passed a b64 encoded image.
+                    is_file = False
+                if is_file:
```
Comment (davehunt) [inline pytest_html/plugin.py]: I'm okay with this patch, but the `is_file` variable name is misleading. The variable determines if the content is a URI or a path, and not specifically if it's a file. Perhaps we could use a better variable name, or change the logic to determine if the content is a base64 encoded string only falling back to creating a link when it's not.

**Comment by davehunt** [comment]
> The Node.js tests didn't run because of a problem during PhantomJS installation. This is unrelated to the proposed change. Please advise.

If anyone has time to investigate that issue it would be appreciated. I had hoped to be home by now but my travel has been disrupted so I may not get time to look for a few days at best. I wouldn't hold up this patch though, if the failure is clearly unrelated.

**Comment by davehunt** [comment]
I've fixed the Travis CI failure in 764ef3cbbe1dfbe22587ada4856fd67c78603558 so once the requests changes have been made I'm happy to merge and release a new version. Thanks @pinkie1378 and @nicoddemus for your contributions.

**Comment by davehunt** [comment]
pytest-html [1.16.1](https://pypi.python.org/pypi/pytest-html/1.16.1) has been released with this fix.

**Pull request merged**